src/__experimental__/components/checkable-input/checkable-input.style.js

```diff
@@ -56,12 +58,42 @@ const StyledCheckableInputWrapper = styled.div`
       }
     `}
 
-    ${fieldHelpInline && `
+    ${fieldHelpInline && css`
+      ${FieldLineStyle} {
+        flex-wrap: nowrap;
+      }
+
+      ${StyledCheckableInput} {
+        margin-right: 0;
+        margin-left: 8px;
+      }
+      
       ${FieldHelpStyle} {
-        display: inline;
+        flex-grow: 0;
+        flex-basis: auto;
         padding-left: 0;
         width: auto;
       }
+
+      ${LabelStyle} {
+        flex: unset;
```

Comment:
`unset` isn't supported by IE: :warning:

* https://developer.mozilla.org/en-US/docs/Web/CSS/unset#Browser_compatibility
* https://caniuse.com/#feat=css-unset-value

Reply:
IE 😠

Reply:
removed

Reply:
Thanks for removing that in b4ea70b :white_check_mark: